gcloud/connection.py

```diff
@@ -219,6 +225,10 @@ def _do_request(self, method, url, headers, data):
         :type data: string
         :param data: The data to send as the body of the request.
 
+        :type dummy: object or :class:`NoneType`
+        :param dummy: Unused ``target_object`` here but may be used
+                      by a superclass.
```

Comment:
@tseaver Comment from cbabbb1b44ac87e84b7932adc1b3e87a711c4ecf:

> Shouldn't the name match? And I think this should be "may be used by a subclass".

---

Do you mean `dummy` should match the other uses or the `target_object` in the docstring is non-matching?

I used `dummy` here to make `pylint` happy (and for no other reason).